gefserver/pier/staging.go

```diff
@@ -100,14 +100,22 @@ func (p *Pier) ListFiles(volumeID db.VolumeID, filePath string) ([]VolumeItem, e
 		return volumeFileList, def.Err(err, "running image failed")
 	}
 
-	// Reading the JSON file
-	volumeFileList, err = p.readJSON(string(containerID), "/root/_filelist.json")
-	if err != nil {
-		return volumeFileList, def.Err(err, "readJson failed")
+	// Wait till the container stops and after that read the JSON file
+	for {
+		cont, err := p.docker.client.InspectContainer(containerID)
```

Comment:
waitcontainerorswarmservice removes the container or a service after it is stopped